src/helpers/utils.ts

```diff
@@ -5,3 +5,12 @@
  */
 export const minifyHTML = (html: string) =>
   html.replace(/\n/g, "").replace(/  /g, "");
+
+export const formatBytes = (bytes: number, decimals: number = 2) => {
+  if (bytes === 0) { return '0 Bytes'; }
+  const k: number = 1024;
```

Comment:
Personally, I'd prefer variables to named, k, dm and i don't mean much and webpack minifies this anyway, so would be better if it was human readable IMO.